staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/conversion/metrics.go

```diff
@@ -86,3 +87,68 @@ func (m *converterMetric) Convert(in runtime.Object, targetGV schema.GroupVersio
 	}
 	return obj, err
 }
+
+type WebhookConversionErrorType string
+
+const (
+	WebhookConversionCallFailure                   WebhookConversionErrorType = "webhook_conversion_call_failure"
+	WebhookConversionMalformedResponseFailure      WebhookConversionErrorType = "webhook_conversion_malformed_response_failure"
+	WebhookConversionPartialResponseFailure        WebhookConversionErrorType = "webhook_conversion_partial_response_failure"
+	WebhookConversionInvalidConvertedObjectFailure WebhookConversionErrorType = "webhook_conversion_invalid_converted_object_failure"
+	WebhookConversionNoObjectsReturnedFailure      WebhookConversionErrorType = "webhook_conversion_no_objects_returned_failure"
+)
+
+var (
+	Metrics = newWebhookConversionMetrics()
+)
+
+// WebhookConversionMetrics instruments webhook conversion with prometheus metrics.
+type WebhookConversionMetrics struct {
+	webhookConversionRequest *metrics.CounterVec
+	webhookConversionLatency *metrics.HistogramVec
+}
+
+func newWebhookConversionMetrics() *WebhookConversionMetrics {
+	webhookConversionRequest := metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Name:           "webhook_conversion_requests",
```

Comment:
1) Subsytem should be "apiserver"
2) Counters should be suffixed `_total`.

Reply:
Do we also provide "Namespace = apiextensions-apiserver" ?

Reply:
No, I'd just use "apiserver" as the Namespace. Otherwise the metric name with be prefixed `apiserver_apiextensions_apiserver_"

Reply:
Not clear.
you mean both subsystem and namespace will be "apiserver" or have to just provide namespace without subsystem ?

Making both apiserver will create name like "apiserver_apiserver_webhook_conversion_duration_seconds"

Reply:
The metric name is comprised as `<Namespace>_<Subsystem>_<Name>`. So if you specify  "apiextensions-apiserver" as a namespace and "apiserver" as a subsystem, you end up with `apiserver_apiserver_apiextensions` as a prefix to your metric name. 

I'm just saying only use oneof {Namespace,Subsystem}, do not use both. And use "apiserver", since that's what we use everywhere else.